torch/nn/modules/adaptive.py

```diff
@@ -275,13 +275,13 @@ def log_prob(self, input: Tensor) -> Tensor:
             - Output: :math:`(N, \texttt{n\_classes})`
 
         """
-
         head_output = self.head(input)
         return self._get_full_log_prob(input, head_output)
 
     def predict(self, input: Tensor) -> Tensor:
-        r""" This is equivalent to `self.log_prob(input).argmax(dim=1)`,
-        but is more efficient in some cases.
+        r"""Return the class with the highest probability for each example in the input minibatch.
+
+        This is equivalent to `self.log_prob(input).argmax(dim=1)`, but is more efficient in some cases.
```

Comment:
nit: similarly not your fault but could we fix
```suggestion
        This is equivalent to ``self.log_prob(input).argmax(dim=1)``, but is more efficient in some cases.
```